.gitignore

```diff
@@ -5,3 +5,4 @@ gradle.properties
 out/
 rewrite-java/src/main/gen/
 .idea/
+rewrite.iml
```

Comment:
This is an artifact of the old IntelliJ installation that you had. No newer IntelliJ versions should generate this file.

Reply:
@jkschneider thanks, I removed it